logstash/filters/bytes2human.rb

```diff
@@ -42,7 +42,7 @@ def filter(event)
   def convert(event)
     @convert.each do |field, direction|
       next unless event.include?(field)
-      original = event[field]
+      original = event.get(field)
```

Comment:
might want to add this to upstream too https://github.com/robin13/logstash-filter-bytes2human